libindy/include/indy_non_secrets.h

```diff
@@ -257,6 +257,113 @@ extern "C" {
                                                                       indy_error_t  err)
                                                 );
 
+    /// Gets schema json data for specified schema id.
+    /// If data is present inside of cache, cached data is returned.
+    /// Otherwise data is fetched from the ledger and stored inside of cache for future use.
+    ///
```

Comment:
please add EXPERIMENTAL notice  here and everywhere in new API (headers, rust API, wrappers)